federation/apis/openapi-spec/v1beta1.extensions.json

```diff
@@ -0,0 +1,104 @@
+{
+  "swagger": "2.0",
+  "info": {
+   "title": "Generic API Server /apis/extensions",
+   "version": "unversioned"
+  },
+  "paths": {
+   "/apis/extensions/": {
```

Comment:
There were a bug in generation code fixed by #34150, I will regenerate these after the PR is merged.